libs/librtlnumber/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@ cmake_minimum_required(VERSION 3.9)
 
 project("librtlnumber")
 
+option(RTL_ALLOW_UNKNOWN_COMPARE "Skips initial check for unknowns in comparison and compares MSB-to-LSB" OFF)
+
+if(RTL_ALLOW_UNKNOWN_COMPARE)
+    add_definitions(-DRTL_ALLOW_UNKNOWN_COMPARE)
+endif()
+
```

Comment:
I think it is better off being a runtime variable (or at least template)
Although it will change a few of the call spot in Odin, if we are to consume it with the simulator and the synthesizer, they have to use it differently. Since the simulator is an integral part of the tool (than can and should be changed eventually) We cannot compile this both way.

Reply:
If you think that would be valuable I can. Personally, I can't see a use case where we would want it compiled in both situations.

Reply:
I'm not sure I understand what you mean by needing to consume it differently in synthesizer and simulator?

Reply:
I don't think it will be an issue for the near future so I can merge.